src/Apps/ArtQuiz/artQuizRoutes.tsx

```diff
@@ -1,8 +1,26 @@
 import loadable from "@loadable/component"
-import { RedirectException } from "found"
+import { Match, RedirectException } from "found"
 import { graphql } from "react-relay"
+import { ArtsyResponse } from "Server/middleware/artsyExpress"
 import { AppRouteConfig } from "System/Router/Route"
 
+const LOGIN_COPY = "Log in to take the Art Quiz."
+const REDIRECT_URL = `/login?redirectTo=/art-quiz&afterSignUpAction=/art-quiz&copy=${LOGIN_COPY}`
```

Comment:
`afterSignUpAction=/art-quiz` isn't a valid afterSignUpAction, you can remove that param.